datacube/scripts/user.py

```diff
@@ -17,16 +26,94 @@
 def user_cmd():
     pass
 
+def build_user_list(index):
+    lstdct = []
+    for role, user, description in index.users.list_users():
+        info = OrderedDict((
+            ('role', role),
+            ('user', user),
+            ('description', description)
+        ))
+        lstdct.append(info)
+    return lstdct
+
+def _write_csv(index):
+    writer = csv.DictWriter(sys.stdout, ['role', 'user', 'description'], extrasaction='ignore')
+    writer.writeheader()
+
+    #for role, user, description in index:
+        #click.echo('{0:6}\t{1:15}\t{2}'.format(role, user, description if description else ''))
```

Comment:
Please remove any commented out code. It ends up being really confusing for anyone reading the code in the future.